recipes/_data/city_boro.csv

```diff
@@ -0,0 +1,50 @@
+city,boro
```

Comment:
This csv is brought into cover any addresses in queens where the neighborhood name is in the address rather than the borough and is meant to improve the geocoding?

Reply:
yup! sorry I forgot to describe this in the PR description, just added it

we were using zipcode before but it looks like the source data has zipcode errors (manually entered) but the "city" doesn't

also, the previous approach of "call everything Queens that isn't the other 4 boroughs" meant were were trying to geocode everything in NY state. some address (e.g. actually in Albany) were erroneously geocoding to Queens because the street address exists in both cities

Reply:
Just to clarify, it seems now based on the csvs that we're now just going with missing zip codes -> assume in Queens? Where before, any entry not in the other 4 boroughs-even if they had a zip code from Albany-were being labeled as Queens?

Reply:
@fvankrieken your right about the old method

but the new method doesn't use zip code at all. it's:
- join `city_boro.csv` to the data using `city` and `facility_city` respectively
- rename `boro` (value from the csv) to `borough`
- geocode records (geocoding requires a `borough`)

so for all records, the assigned borough is based on it's city and a missing zip code has no effect

Reply:
Ah gotcha - I was looking [here](https://github.com/NYCPlanning/ceqr-app-data/pull/154/files#diff-2e30ac394ca79cc3d48af2bf5efa838eac9dc7c2acf241a78db0c1bb25e94215R171), but looks like you [deleted the only reference to that file](https://github.com/NYCPlanning/ceqr-app-data/pull/154/files#diff-a0522be4186dd2d8352870cd30783ae0b71f7335e29322746ed3f36ba4c2ae3fR43). Can that csv just be deleted? Or saving it to posterity. Or am I missing another reference somewhere

Reply:
could be deleted but seemed worth save for posterity